cli/tests/integrations/test_package.py

```diff
@@ -301,7 +301,11 @@ def test_describe_specific_version():
 
 def test_bad_install():
     args = ['--options=tests/data/package/chronos-bad.json', '--yes']
-    stdout = b""
+    stdout = (
+        b'We recommend a minimum of one node with at least 1 '
+        b'CPU and 2GB of RAM available for the Chronos '
+        b'Service.\n'
+        b'Installing Marathon app for package [chronos] version [2.4.0]\n')
```

Comment:
The last line here is new, because I removed [this line](https://github.com/dcos/dcos-cli/pull/729/files#diff-1649767c70a587014cbfef7156f395edL370). But otherwise the behavior is the same as before, with one less request made to Cosmos.